packages/astro/src/events/session.ts

```diff
@@ -121,15 +120,12 @@ export function eventCliSession(
 	const payload: EventCliSessionInternal = {
 		cliCommand: event.cliCommand,
 		// Versions
-		astroVersion: event.astroVersion,
 		viteVersion: getViteVersion(),
 		nodeVersion: process.version.replace(/^v?/, ''),
 		configKeys: userConfig ? configKeys(userConfig, '') : undefined,
 		// Config Values
 		config: configValues,
 		flags: cliFlags,
-		// Optional integrations
-		optionalIntegrations: userConfig?.integrations?.length - integrations?.length,
```

Comment:
this was added as a "nice-to-have" in a PR that was fixing a different bug, but I don't think this is actually valuable to us so I removed it.